src/components/RubberStampOverlay/RubberStampOverlay.scss

```diff
@@ -3,43 +3,195 @@
 
 .rubber-stamp-overlay {
   margin-left: 0px;
-  margin-right: 2px;
   margin-bottom: 8px;
+  padding-left: 8px;
+  padding-right: 4px;
 
   @include mobile {
     margin-bottom: 0px;
     margin-right: 0px;
     padding-top: 12px;
   }
+  .custom-stamp-panel {
+    max-height: 270px;
+    overflow: auto;
+    margin-bottom: 8px;
 
-  .default-stamps-container {
+  }
+  .standard-stamp-panel {
     display: flex;
+    align-items: center;
     flex-direction: row;
     flex-wrap: wrap;
-    height: 300px;
+    flex-flow: row wrap;
+    align-content: flex-start;
+    max-height: 300px;
     overflow-y: scroll;
-    padding-right: 14px;
-
+  }
+  .standard-stamp-panel {
     @include mobile {
       height: 236px;
-      padding-right: 30px;
       width: 300px;
     }
+  }
+  .custom-stamp-panel {
+    @include mobile {
+      height: 188px;
+    }
+  }
+  .rubber-stamp {
+    border-radius: 4px;
+    flex-basis: 50%;
+    cursor: pointer;
 
-    .rubber-stamp {
-      border-radius: 4px;
-      flex-basis: 50%;
-      cursor: pointer;
+    &:hover {
+      background: var(--popup-button-hover);
+    }
+
+    img {
+      width: 100%;
+    }
+  }
+
+  .stamp-row {
+    display: flex;
+    flex-direction: row;
+    height: 45px;
+    justify-content: space-between;
+    padding-bottom: 4px;
+
+    @include mobile {
+      justify-content: center;
+    }
+    .icon {
       display: flex;
       align-items: center;
+      justify-content: center;
+      width: 34px;
+      height: 45px;
+      border-radius: 4px;
+      cursor: pointer;
+
+      &:hover {
+        background: var(--tools-overlay-button-hover);
+      }
+    }
+
+
+  }
+  .stamp-row-content {
+    width: 162px;
+    border-radius: 4px;
+    padding: 2px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    cursor: pointer;
+    margin-right: 8px;
+
+    img {
+      max-width: 100%;
+      max-height: 100%;
+    }
+    &:hover {
+      background: var(--popup-button-hover);
+    }
```

Comment:
can you change this to:
```    
&:not(.add-btn) {
      &:hover {
        background: var(--popup-button-hover);
      }
    }
```
just so it is consistent with the add signature button.